core/runtime/c/wrapper.c

```diff
@@ -430,55 +460,60 @@ int main(int argc, char **argv) {
         close(output[0]);
         close(input[1]);
 
-        print(DEBUG, "Wait completion of scontainer stage1");
+        singularity_message(DEBUG, "Wait completion of scontainer stage1\n");
         if ( wait(&status) != stage1 ) {
-            pfatal("Can't wait child");
+            singularity_message(ERROR, "Can't wait child\n");
+            exit(1);
         }
 
         if ( WIFEXITED(status) || WIFSIGNALED(status) ) {
             if ( WEXITSTATUS(status) != 0 ) {
-                pfatal("Child exit with status %d", WEXITSTATUS(status));
+                singularity_message(ERROR, "Child exit with status %d\n", WEXITSTATUS(status));
+                exit(1);
             }
         }
         close(stage_socket[0]);
```

Comment:
Can you remove this line ? It's an oversight on my side because it can close stdin/stdou/stderr depending of initialization value. Thanks

Reply:
I fixed `MESSAGELEVEL` issue, but not sure exactly which line to remove here. Can you submit PR on top of this with just that small change?

Reply:
Done